### ISSUE TITLE
why keras.layers.MultiHeadAttention q-dim don't / heads?

### ISSUE THREAD

**Issue by zhanlie2008** [keras-team/keras-io#902]

**Comment by pcoet** [comment]
Hi @zhanlie2008, thanks for reaching out. The issue backlog is not the best place for support requests. Can you repost this to one of the forums below? That way your question will have more visibility.

* [TensorFlow forum](https://discuss.tensorflow.org/)
* [Keras mailing list](https://groups.google.com/forum/#!forum/keras-users)

Thanks!
